src/scenes/MainMenu.ts

```diff
@@ -33,6 +34,9 @@ export class MainMenu extends Scene {
 
         // Create orientation guide
         this.orientationGuide = new OrientationGuide(this);
+        
+        // Add logos to the top right corner
+        Logos.get(this);
```

Comment:
to me get shouldn't really .... change things
it doesn't seem like we are using the singleton, so do you want to just do new Logos(this)?